tests/components/cloud/test_init.py

```diff
@@ -137,6 +137,14 @@ async def test_on_connect(hass, mock_cloud_fixture):
 
     assert len(hass.states.async_entity_ids("binary_sensor")) == 0
 
+    all_events = []
+
+    @callback
+    def capture_events(ev):
+        all_events.append(ev.event_type)
+
+    hass.bus.async_listen(cloud.EVENT_CLOUD_CONNECTED, capture_events)
```

Comment:
We have this helper:
https://github.com/home-assistant/core/blob/caa5578134bb8d31ef71c56ae8c75480dc2275e6/tests/common.py#L1156